omnibus/cookbooks/omnibus-supermarket/recipes/cookbook_test.rb

```diff
@@ -26,13 +26,13 @@
 package_path = node['supermarket']['test']['package_path'] || '/tmp/packages'
 version_to_install = node['supermarket']['test']['version_to_install'] || 'nope_set_VERSION_TO_INSTALL'
 
-package 'supermarket' do
-  case node['platform_family']
-  when 'debian'
-    provider Chef::Provider::Package::Dpkg
+case node['platform_family']
+when 'debian'
+  dpkg_package 'supermarket' do
     source "#{package_path}/supermarket_#{version_to_install}-1_amd64.deb"
-  when 'rhel'
-    provider Chef::Provider::Package::Rpm
+  end
+when 'rhel'
+  rpm_package 'supermarket' do
```

Comment:
Foodcritic didn't like `package` with a `provider` property and said "Use the specific package resource!"

Gotta use the specific `{dpkg|rpm}_package` resources because the `package` resource will default to `{apt|yum}_package` which (at least `apt_package`) don't accept a `source` property. See https://github.com/chef/supermarket/pull/1768#discussion_r238826986 for a previous review of this change.